<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iconograph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="canvas">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabs">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="1" column="5">
         <widget class="QCheckBox" name="portraitCheck">
          <property name="text">
           <string>Portrait</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="gainLabel">
          <property name="text">
           <string>Gain</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QCheckBox" name="colourCheck">
          <property name="text">
           <string>Force Bayer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="exposureLabel">
          <property name="text">
           <string>Exposure</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="exposureSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="flashLabel">
          <property name="text">
           <string>Flash</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="flashCheck">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="flashSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="flashSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="exposureCheck">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="exposureSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="singleStep">
           <number>8</number>
          </property>
          <property name="pageStep">
           <number>32</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="adcSlider">
          <property name="maximum">
           <number>7</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="adcLabel">
          <property name="text">
           <string>ADC Reference</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="gainSlider">
          <property name="minimum">
           <number>16</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
          <property name="pageStep">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSpinBox" name="adcSpin">
          <property name="maximum">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="gainSpin">
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <number>16</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="gainCheck">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QCheckBox" name="compandingCheck">
          <property name="text">
           <string>Companding</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="adcReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advancedTab">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableView" name="advancedTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="status">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>flashSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>flashSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flashSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>flashSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flashCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>flashSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flashCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>flashSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>exposureSlider</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>exposureSpin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>gainSpin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>gainSlider</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adcSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>adcSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adcSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>adcSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
